doc/docs/server-pull.md

```diff
@@ -76,6 +77,12 @@ The [`clientID`](api/classes/replicache#clientid) of the requesting Replicache i
 
 The cookie that was received last time a pull was done. `null` if this is the first pull from this client.
 
+### `lastMutationID`
+
+The `lastMutationsID` of this client's base snapshot. This means that we have
+successfully applied the local mutations up to (and including) `lastMutationID`
+from this client on the server.
```

Comment:
Not really clear to me what to document here

Reply:
```suggestion
The `lastMutationID`  the client received for the last [pull response](https://doc.replicache.dev/server-pull#http-request-body). This can be useful in cases where a server receives a pull request from a client it doesn't know about (perhaps because the client state has been deleted). In that case an easy thing to do is to re-establish the record of the client on the server side with the `lastMutationID` it is expecting, which is this value.
```